comp/venue/insurance.yaml

```
@@ -1,12 +1,20 @@
 summary: Acquire public liability insurance for the competition
 
-priority: major
+priority: should
```

Insurance sounds pretty important
```suggestion
priority: must
```